app/controllers/solidus_klarna_payments/callbacks_controller.rb

```diff
@@ -1,29 +0,0 @@
-# frozen_string_literal: true
-
-module SolidusKlarnaPayments
```

Comment:
Instead of just removing these controllers, what do you think to just deprecate them for now?